--- 
layout: post 
title: Liquid Biopsies and Acquired Resistance
tags:
- cancer
- resistance
- liquid-biopsy
--- 

Open any standard cell biology textbook, and you are likely to see a similar diagram describing a cell and its various compartments:  DNA coiled into chromosomes, chromosomes tightly packed into the nucleus.  The entire cell neatly enclosed in a cellular membrane.  According to the model, DNA is cell-bound, uniquely attached to and anchored within a protective and atomic cellular unit.

<div class="photo-left">
	<a href="https://raw.github.com/ecerami/ecerami.github.io/master/img/liquid_biopsy.jpg"><img src="https://raw.github.com/ecerami/ecerami.github.io/master/img/liquid_biopsy.jpg"></a>
	<p>A group from the Cancer Research UK has successfully applied whole exome sequencing to serial liquid biopsies to identify causes of acquired resistance in multiple cancer types.  Image from:  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23563269">M Murtaza, et al.</a>  Nature. 2013 May 2;497(7447):108-12.</p>
</div>

Except, of course, when it's not.  In 1948, two researchers P. Mandel and P. Metais made the startling discovery that human blood contains free-floating or cell-free DNA [1].  This "naked" DNA, now dubbed circulating free DNA or cfDNA exists is all of us, but researchers have since discovered that certain disease conditions, including rheumatoid arthritis, pancreatitis and cancer can result in increased levels of cfDNA.  Where and how this naked DNA originates is still an active area of research, but most now believe it is the result of dying cells releasing their DNA into surrounding tissues.  There is also evidence, although no clearly defined mechanism, by which living cells actively shed DNA into circulation [2].

Combine the fact that tumors release naked DNA into the bloodstream with advances in next-generation sequencing, and we now have a new era of "liquid biopsies".  Unlike traditional biopsies, liquid biopsies have the advantage of being non-invasive (no surgery required), and also have the potential to capture multiple sties of tumor growth all at once.  Theoretically, liquid biopsies therefore capture both tumor heterogeneity across multiple sites, and tumor evolution across multiple time points.

Multiple applications are on the horizon for liquid biopsies, including the early detection of cancer, and early detection of cancer recurrence.  The one area that has seen the greatest advance though is the use of liquid biopsies to identify the cause of acquired resistance, in response to targeted therapy.  Every known targeted therapy results in some form of acquired resistance.  Bert Vogelstein of Johns Hopkins University has famously called acquired resistance a "fait accompli" [3] -- even before targeted therapy begins, a pool of tumor cells already contain the very mutations required to circumvent the therapy, and it is just a matter of time before these cells are selected for, and assert themselves.

<div class="sidebar-right">
	<p><b>Recommended Reading:</b>
	<p>
		<ul>
			<li>Diaz LA Jr, et al., <a href="http://www.ncbi.nlm.nih.gov/pubmed/22722843">The molecular evolution of acquired resistance to targeted EGFR blockade in colorectal cancers.</a>  Nature. 2012 Jun 28;486(7404):537-40.</li>
			<li>M Murtaza, et al., <a href="http://www.ncbi.nlm.nih.gov/pubmed/23563269">Non-invasive analysis of acquired resistance to cancer therapy by sequencing of plasma DNA.</a>  Nature. 2013 May 2;497(7447):108-12.</li>
			<li>Emily Crowley, et al., <a href="http://www.ncbi.nlm.nih.gov/pubmed/23836314">Liquid biopsy:  monitoring cancer-genetics in the blood.</a>  Nature Reviews Oncology, July 9, 2013.
			</li>
		</ul>
</div>

The promise, of course, is that one could apply a targeted therapy, use serial liquid biopsies to determine and identify the cause of acquired resistance, and then apply a newly refined targeted therapy.  Or, do this with enough patients to identify specific trends in resistance, and apply multiple targeted therapies in combination in anticipation of acquired resistance.

Last year, L. Diaz et al. from Johns Hopkins University reported one of the most successful applications of liquid biopsies to the study of acquired resistance [4].  The study focused on identifying the mechanism of resistance to panitumumab, an anti-EGFR antibody treatment used to treat colorectal cancer.  The study followed 24 KRAS wild-type patients receiving panitumumab therapy, and liquid biopsies were obtained and analyzed every four weeks.  Only common mutations at codons 12 and 13 of KRAS and codons 600 and 601 in BRAF were assessed.  Nine of 24 patients (38%), were found to develop resistance mutations in KRAS, and even more significantly, 3 of the 9 patients showed KRAS resistance before any radiographic evidence of disease progression.

Fast forward one year, and a group from Cancer Research UK has published another more extensive study of liquid biopsies and resistance in multiple cancer types, including breast, ovarian and lung [5].  In contrast to the Hopkins study, however, which focused exclusively on a small set of mutations in just two genes, the Cancer Research UK study performed whole exome sequencing on serial liquid biopsies.  In doing so, the group was able to identify multiple mechanisms of resistance -- for example, development of activating PIK3CA mutations in a breast cancer patient being treated with paclitaxel; and an activating EGFR T790M mutation in a non-small cell lung cancer patient being treated with gefitinib.

The Hopkins and Cancer Research UK studies are proof-of-concept studies, and there are as yet no standard protocols for obtaining and analyzing liquid biopsies in the clinical setting.  Nonetheless, the studies do raise the exciting possibility of more wide-spread use of liquid biopsies in clinical trial design and standard clinical practice.

The studies also illustrate just how far the scientific and medical communities have come in the past 65 years since Mandel and Metais first published their initial findings regarding circulating free DNA.  To put their findings into historical context, the two researchers published their findings two years before the development of the ENIAC, the first general purpose computer, and five years before Watson and Crick published the structure of DNA.  Could these two have imagined that 65 years after their initial discovery that a clinician could draw blood from a cancer patient, extract and "read" the individual letters of DNA, transfer the DNA sequence to a tiny computer no bigger than a file folder, and then pinpoint the exact molecular cause of a person's cancer?  Everything about cancer remains complex, daunting and sometimes overwhelming.  But, these are indeed exciting times.

## Footnotes

[1]  Mandel P, Metais P. **Les acides nucleiques du plasma sanguin chez lâ€™homme.** C R Acad Sci Paris 1948;142:241-243.

[2]  For an overview of the origins of cfDNA, see:  Maniesh Van Der Vaart, Piet J. Pretorius, **Circulating DNA:  Its Origin and Fluctuation**, Ann N Y Acad Sci. 2008 Aug;1137:18-26. doi: 10.1196/annals.1448.022. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18837919">PubMed</a>]

[3] Carl Zimmer, **Studying Tumors Differently, in Hopes of Outsmarting Them**, New York Times, June 27, 2013.  [<a href="http://www.nytimes.com/2013/06/27/science/studying-tumors-differently-in-hopes-of-outsmarting-them.html?_r=0">Link</a>]

[4]  Diaz LA Jr, et al., **The molecular evolution of acquired resistance to targeted EGFR blockade in colorectal cancers.**  Nature. 2012 Jun 28;486(7404):537-40.  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22722843">PubMed</a>]

[5]  M Murtaza, et al., **Non-invasive analysis of acquired resistance to cancer therapy by sequencing of plasma DNA.**  Nature. 2013 May 2;497(7447):108-12. [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23563269">PubMed</a>]
